search-app/src/cmr/search/data/complex_to_simple_converters/spatial.clj

```diff
@@ -156,6 +157,13 @@
                 (crossing-ranges->condition crossings)])))
           lat-ranges-crossings))))
 
+(defn- spatial-operator
+  "Returns the operator used to determine if spatial queries must match all polygons."
+  [params]
+  (if (= (type params) clojure.lang.PersistentArrayMap)
```

Comment:
is there a specific reason `(map?)` isn't used?

Reply:
No good reason. Pushed a change to simplify it to use `(map?)`.